tools/google-cloud-support-slackbot/README.md

```diff
@@ -0,0 +1,153 @@
+
+Go to [Slack Apps](http://api.slack.com/apps) to do the following:
+
+1. Click **Create New App** and select **From scratch**. Name your app `Google Cloud Support Bot` and select your workspace
+2. Under **Settings > Basic Information**, scroll down to **Display Information** and upload the [google_cloud_support_buddy_big.png](google_cloud_support_buddy_big.png) or an icon of your choosing.
+3. Go to **Features > Slash Commands** and create the following command:
+	1. Command: `/google-cloud-support `
+	1. Request URL: `http://<your_vm_external_ip>/google-cloud-support`
+	1. Short description: `Track and manage your Google Cloud support cases in Slack. Use /google-cloud-support help for the list of commands`
+	1. Usage Hint: `[command] [parameter 1] [parameter 2] [parameter 3]`
+4. Go to **Features > OAuth & Permissions**. Scroll down to **Scopes** and add the **chat:write** scope. Add the **commands** scope if it isn't listed already listed
+5. At the top of the **Features > OAuth & Permissions** page, under **OAuth Tokens for Your Workspace**, click **Install to Workspace**. Copy the token. You may need Slack admin approval to install the app.
+
+## Setup Part 4 - Google Cloud Phase 2
+
+Return to [Google Cloud](https://cloud.google.com/console) and from **Compute Engine > VM instances**, perform the following:
+
+1. SSH into the VM that you created in part 2 of this setup guide
+2. Run the following commands:
+	1. `sudo apt-get update`
+	1. `sudo apt-get -y install subversion`
+	1. `sudo apt-get -y install python3-pip`
+	1. `sudo apt-get -y install nginx`
+	1. `cd /`
+	1. `sudo svn export https://github.com/GoogleCloudPlatform/professional-services/trunk/tools/google-cloud-support-slackbot`
+	1. `cd /google-cloud-support-slackbot`
+	1. Use sudo to open the `default` file with your editor of choice, and replace <STATIC_IP> with the external ip address of your VM. Then save and close the file
+	1. `sudo mv default /etc/nginx/sites-available/`
+	1. Use sudo to open the `.env` file with your editor of choice. Enter your API Key, Slack Token, and numeric org id in their respective locations. Then save and close the file
+	1. `sudo chmod +x google_cloud_support_slackbot.py`
+3. Close the SSH session
+4. From Compute Engine > VM instances, click your VM name to go to your VM instance details
+5. Stop the VM
+6. Once the VM is stopped, click the 'EDIT' button
+7. Scroll down to the Custom metadata section and add the following key-value pair:
+	1. key: `startup-script`
+	1. value:  
+	`cd /google-cloud-support-slackbot`  
+	`pip3 install -r requirements.txt`   
+	`/google-cloud-support-slackbot/google_cloud_support_slackbot.py`
+8. Scroll to the bottom of the page and click 'Save'
+9. Start your VM
+
+## Closing
+
+With that you should be all setup! And as a reminder, if you had to create the SSH firewall rule, it is recommended that you go back and disable it. If you ever need to SSH into the machine you can always enable the rule again as needed.
```

Comment:
Can we guide user how to perform verification of e2e communication?

Reply:
Added!